wire/error.go

```diff
@@ -17,19 +164,56 @@ import (
 // differentiate between general io errors such as io.EOF and issues that
 // resulted from malformed messages.
 type MessageError struct {
-	Func        string // Function name
-	Description string // Human readable description of the issue
+	Op          string    // The error source
```

Comment:
This is a breaking change to the API.  It would need to have the major bumped and everything else that uses it if you do this.  I *highly* advise against it.  You can add to the struct, but you can't change the publicly exported name without a major API bump.

Reply:
Oops, rectifying.